src/crypto/chacha_poly_aead.cpp

```diff
@@ -27,20 +26,53 @@ int timingsafe_bcmp(const unsigned char* b1, const unsigned char* b2, size_t n)
 
 #endif // TIMINGSAFE_BCMP
 
-ChaCha20Poly1305AEAD::ChaCha20Poly1305AEAD(const unsigned char* K_1, size_t K_1_len, const unsigned char* K_2, size_t K_2_len)
+ChaCha20Forward4064::ChaCha20Forward4064(const Span<unsigned char> key)
```

Comment:
If your intent is to have the contents of the Span const (which I think it is), this should be `Span<const unsigned char>`.

Reply:
Done across the board.